packages/protocol/test/governance/election.ts

```diff
@@ -0,0 +1,854 @@
+    })
+
+    it('should emit the MaxNumGroupsVotedForSet event', async () => {
+      const resp = await election.setMaxNumGroupsVotedFor(newMaxNumGroupsVotedFor)
+      assert.equal(resp.logs.length, 1)
+      const log = resp.logs[0]
+      assertContainSubset(log, {
+        event: 'MaxNumGroupsVotedForSet',
+        args: {
+          maxNumGroupsVotedFor: new BigNumber(newMaxNumGroupsVotedFor),
+        },
+      })
+    })
+
+    it('should revert when the maxNumGroupsVotedFor is unchanged', async () => {
+      await assertRevert(election.setMaxNumGroupsVotedFor(maxNumGroupsVotedFor))
+    })
+
+    it('should revert when called by anyone other than the owner', async () => {
+      await assertRevert(
+        election.setMaxNumGroupsVotedFor(newMaxNumGroupsVotedFor, { from: nonOwner })
+      )
+    })
+  })
+
+  describe('#markGroupEligible', () => {
+    const group = accounts[1]
+    describe('when the group has members', () => {
+      beforeEach(async () => {
+        await mockValidators.setMembers(group, [accounts[9]])
+      })
+
+      describe('when the group has no votes', () => {
+        let resp: any
+        beforeEach(async () => {
+          resp = await election.markGroupEligible(NULL_ADDRESS, NULL_ADDRESS, { from: group })
+        })
+
+        it('should add the group to the list of eligible groups', async () => {
+          assert.deepEqual(await election.getEligibleValidatorGroups(), [group])
+        })
+
+        it('should emit the ValidatorGroupMarkedEligible event', async () => {
+          assert.equal(resp.logs.length, 1)
+          const log = resp.logs[0]
+          assertContainSubset(log, {
+            event: 'ValidatorGroupMarkedEligible',
+            args: {
+              group,
+            },
+          })
+        })
+
+        describe('when the group has already been marked eligible', () => {
+          it('should revert', async () => {
+            await assertRevert(
+              election.markGroupEligible(NULL_ADDRESS, NULL_ADDRESS, { from: group })
+            )
+          })
+        })
+      })
+    })
+
+    describe('when the group has no members', () => {
+      it('should revert', async () => {
+        await assertRevert(election.markGroupEligible(NULL_ADDRESS, NULL_ADDRESS, { from: group }))
+      })
+    })
+  })
+
+  describe('#markGroupIneligible', () => {
+    const group = accounts[1]
+    describe('when the group is eligible', () => {
+      beforeEach(async () => {
+        await mockValidators.setMembers(group, [accounts[9]])
+        await election.markGroupEligible(NULL_ADDRESS, NULL_ADDRESS, { from: group })
+      })
+
+      describe('when called by the registered Validators contract', () => {
+        let resp: any
+        beforeEach(async () => {
+          await registry.setAddressFor(CeloContractName.Validators, accounts[0])
+          resp = await election.markGroupIneligible(group)
+        })
+
+        describe('when the group has votes', () => {})
```

Comment:
Empty `describe`. Did you mean to put the `it`s below inside of it?

Reply:
Nope, not sure where this came from